packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -57,6 +57,16 @@
 	font-size: 11px !important;
 }
 
+.edit-site-global-styles-section-title {
+	font-size: calc(1.95 * 8px);
+	color: rgb(30, 30, 30);
```

Comment:
How come we are using `rgb` notation here?

Reply:
I think it is copy pasta, I can check out what variable serves me.